internal/validation/validation_utils.go

```diff
@@ -16,13 +16,22 @@ import (
 )
 
 func ValidatePassword(v interface{}, k string) (warnings []string, errors []error) {
+	var specialSymbols = []rune{'\'', '!', '"', '#', '$', '%', '&', '(', ')', '*', '+', '-', '.', '/', ':', ';', '<', '=', '>', '?', '@', '[', '\\', ']', '^', '_', '`', '{', 'Ι', '}', '~'}
+	return validatePasswordInternal(v, k, specialSymbols, 8)
+}
+
+func ValidateNsxEdgePassword(v interface{}, k string) (warnings []string, errors []error) {
```

Comment:
Can you, please, add unit test for this method.

Reply:
Done